protocol/src/traits/ibc.rs

```diff
@@ -1,14 +1,183 @@
+use cosmos_ibc::core::ics24_host::validate::validate_identifier;
+use cosmos_ibc::core::ics24_host::{path, Path as IbcPath};
+
 use crate::traits::IbcCrossChainStorage;
 use crate::types::{ConsensusStateWithHeight, Header, Metadata};
 use crate::{async_trait, ProtocolResult};
+use std::{
+    fmt::{Display, Formatter},
+    ops::Deref,
+    str::{from_utf8, FromStr},
+};
+#[derive(Debug, Ord, PartialOrd, Eq, PartialEq, Clone)]
+pub struct Error(String);
```

Comment:
Where is this error used?

Reply:
just to simplify the deleted `flex-error`